DatabaseObjects/ScrumMember.java

```diff
@@ -0,0 +1,54 @@
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class ScrumMember implements IDatabaseObject {
+    private int scrumId;
+    private int employeeId;
+    public Employee employee;
+
+    public ScrumMember(int scrumId, int employeeId) {
+        this.scrumId = scrumId;
+        this.employeeId = employeeId;
+    }
+    
+    public ScrumMember(Employee employee) {
+        this.employee = employee;

```

Comment:
so `this.employee` is only initialized when we call this constructor (and not when using `ScrumMember(int scrumId, int employeeId)`? why?